src/library/scala/math/package.scala

```diff
@@ -58,7 +58,21 @@ package object math {
    *          logarithms.
    */
   def exp(x: Double): Double = java.lang.Math.exp(x)
+
+  /** Returns the natural logarithm of a `double` value.
+   * 
+   * The natural logarithm is the logarithm base Euler's number `e`.
```

Comment:
Do we really need this line?  If you're using logarithms and you don't know what a natural logarithm is, I think there's already a problem.  (And I doubt that Euler's number will help anything.)